log-viewer-webui: `@types/node` is missing from devDependencies.

### Bug

As the title suggests. The dev dep is missing from `package.json` specification, although `package-lock.json` does track the dependency:
https://github.com/y-scope/clp/blob/a37f68c6ff6e122e8b31920560eedec15bc7e451/components/log-viewer-webui/client/package-lock.json#L2310-L2320

We will face build failure after we run [`npm prune`](https://docs.npmjs.com/cli/v10/commands/npm-prune). See the "Reproduction steps" section for another potential issue.

### CLP version

a37f68c6ff6e122e8b31920560eedec15bc7e451

### Environment

```
junhao@GIGABYTE:~/workspace/clp/components/log-viewer-webui/client$ npm --version
10.9.2
junhao@GIGABYTE:~/workspace/clp/components/log-viewer-webui/client$ node --version
v22.14.0
```

### Reproduction steps

```
cd clp
cd components/log-viewer-webui/client

rm -rf ./node_modules
rm package-lock.json

npm i
npm run build
```
->
```
> log-viewer-webui-client@0.1.0 build
> tsc -b && vite build

vite.config.ts:1:18 - error TS2307: Cannot find module 'node:path' or its corresponding type declarations.

1 import path from "node:path";
                   ~~~~~~~~~~~

vite.config.ts:19:37 - error TS2304: Cannot find name '__dirname'.

19             "@common": path.resolve(__dirname, "../common"),
                                       ~~~~~~~~~


Found 2 errors.
```

--- a/components/log-viewer-webui/client/package.json
+++ b/components/log-viewer-webui/client/package.json
@@ -34,6 +34,7 @@
     "zustand": "^5.0.3"
   },
   "devDependencies": {
+    "@types/node": "22.15.31",
     "@types/react": "^19.0.10",
     "@types/react-dom": "^19.0.4",
     "@types/react-syntax-highlighter": "^15.5.13",

